sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -634,7 +634,7 @@ case class HiveTableRelation(
   )
 
   override def computeStats(): Statistics = {
-    tableMeta.stats.map(_.toPlanStats(output, conf.cboEnabled))
+    tableMeta.stats.map(_.toPlanStats(output, conf.planStatsEnabled))
```

Comment:
Oh, @maropu . I'm wondering if the following is better. If someone already is `cboEnabled=true`, this will protect the potential regression due to the new option because the new default value of new option is `false`. How do you think about that?
```scala
- tableMeta.stats.map(_.toPlanStats(output, conf.planStatsEnabled))
+ tableMeta.stats.map(_.toPlanStats(output, conf.cboEnabled || conf.planStatsEnabled))
```

Reply:
Ah, I see. It looks resonable to me, and I'll update.